puppet/modules/kubernetes/manifests/kubelet.pp

```diff
@@ -107,8 +107,13 @@
     $_ca_file = $ca_file
   }
 
+  $os_version = 0 + $facts["operatingsystemrelease"]
```

Comment:
Can this part behave like the following:

- If fact is existing and os_version < 7.5 use `svirt_sandbox_file_t`
- Else use `container_file_t`

Also for comparing the version, can we please use that: https://github.com/jetstack/tarmak/blob/fee1c17ecb78942474047e0cb7815fc1105269f8/puppet/modules/prometheus/manifests/kube_state_metrics.pp#L19

Reply:
That's much nicer. I'd got it working with split & more mess but oh well - wish I'd known about this earlier :man_facepalming: